test/unit/helpers/form_helper_test.rb

```diff
@@ -0,0 +1,78 @@
+require "test_helper"
+
+class FormHelperTest < ActionView::TestCase
+  context "form_errors" do
+    should "return an unordered list with errors when there are errors present" do
+      expected = "<ul class=\"help-block error-block\"><li>One</li><li>Two</li><li>Three</li></ul>"
+
+      assert_equal expected, form_errors(%w[One Two Three])
+    end
+
+    should "return an empty unordered list when there are no errors present" do
+      expected = "<ul class=\"help-block error-block\"></ul>"
+
+      assert_equal expected, form_errors
+    end
+  end
+
+  context "form_group" do
+    setup do
+      object = stub(field_name: "", errors: Hash.new([]))
+      @form = FormBuilder.new("edition", object, self, {})
+      @label = '<div class="form-label"><label for="edition_field_name">Field name</label></div>'
+      @wrapped_field = '<div class="form-wrapper"><input type="text" value="" name="edition[field_name]" id="edition_field_name" /></div>'
+      @error_block = '<ul class="help-block error-block"></ul>'
+    end
+
+    should "return a form-group html structure without: errors, help block, custom label or custom attributes" do
+      output = form_group(@form, :field_name) { @form.text_field(:field_name) }
+
+      assert_equal %(<div class="form-group">#{@label}#{@wrapped_field}#{@error_block}</div>), output
+    end
+
+    should "return a form-group html structure with a help block when help text is supplied" do
+      output = form_group(@form, :field_name, help: "Help block text") { @form.text_field(:field_name) }
+      help_block = '<div class="help-block">Help block text</div>'
+
+      assert_equal %(<div class="form-group">#{@label}#{@wrapped_field}#{@error_block}#{help_block}</div>), output
```

Comment:
I don't think we need to test the whole output again. I imagine `assert_match help_block, output` will suffice.

Same comment applies to following tests